scripts/modules/MsIndustryLinks/templates/data_source/DataSourceWorkflow.psm1

```diff
@@ -338,4 +410,116 @@ function Get-WorkflowGuids {
     return $workflowGuids
 }
 
+function Get-ApiName {
+    param (
+        [Parameter(Mandatory = $true, HelpMessage = "The data source type.")]
+        [string] $DataSourceType
+    )
+
+    switch ($DataSourceType.ToLower()) {
+        "azureblobstorage" {
+            return "azureblob"
+        }
+        "dataverse" {
+            return "commondataservice"
+        }
+        "eventhub" {
+            return "eventhubs"
+        }
+        default {
+            throw "The connection type, $DataSourceType, is not supported."
+        }
+    }
+}
+
+function Get-TransformDataSubflow {
```

Comment:
Are these new methods all logic app specific?
If they are, might be helpful adding a docstring to indicate that. Otherwise, we should get the flow methods to adopt these new methods.

Reply:
No, a couple are for Flow. Since the other functions don't have docstrings, I've followed the same pattern. If that is something that needs to be added going forward, I'd rather have this resolved in a different PR. Otherwise, it'll be inconsistent with the existing functions.